be/src/olap/lru_cache.h

```diff
@@ -322,6 +324,9 @@ class LRUCache {
 
     // Separate from constructor so caller can easily make an array of LRUCache
     void set_capacity(size_t capacity) { _capacity = capacity; }
+    void set_element_count_limit(uint32_t element_count_limit) {
+        _element_count_limit = element_count_limit;
```

Comment:
suggest a name _count_capacity

Reply:
use _element_count_capacity instead of it